one_fm/operations/doctype/contracts/contracts.py

```diff
@@ -1038,6 +1038,7 @@ def renew_contracts_by_termination_date():
     if all_due_contracts:
         
         for each in all_due_contracts:
+            contract_doc = frappe.get_doc("Contracts",each.name)
```

Comment:
@Talleyrand333 check how the same thing done here https://github.com/ONE-F-M/One-FM/blob/6c36da931951d84b5ba10ede67d8859cb8d4c0f2/one_fm/operations/doctype/contracts/contracts.py#L427, Do like this, otherwise we may get a permission issue to the Contracts Date doctype, since it is a child table.